docs/_static/spec/rasa.yml

```diff
@@ -905,14 +957,31 @@ components:
       type: object
       properties:
         name:
-          description: >-
-            Name of the action to be executed.
+          description: Name of the action to be executed.
           type: string
           example: utter_greet
         policy:
+          description: Name of the policy that predicted the action (optional).
```

Comment:
you don't have to add the `optional`. There is a special field for it https://swagger.io/docs/specification/data-models/data-types/#null